assets/bingo/lang/pl_pl.json

```diff
@@ -102,6 +113,17 @@
 	"bingo.lobby.credits.dr_brian_lorgon111.sign.line3": "twórca",
 	"bingo.lobby.credits.dr_brian_lorgon111.sign.line4": "Minecraft Bingo",
 
+	"bingo.lobby.credits.community_and_support.sign.line1": "",
+	"bingo.lobby.credits.community_and_support.sign.line2": "Społeczność",
+	"bingo.lobby.credits.community_and_support.sign.line3": "& Wsparcie",
+	"bingo.lobby.credits.community_and_support.sign.line4": "",
+
+	"bingo.lobby.credits.community_and_support.heading": "Społeczność & Wsparcie",
+	"bingo.lobby.credits.community_and_support.description": "W tej książce znajdują się przydatne linki, dzięki którym możesz uzyskać pomoc techniczną lub zgłosić błędy / zaproponować nowe funkcje.",
+	"bingo.lobby.credits.community_and_support.issue_link": "Dziennik błędów i problemów",
```

Comment:
Google translator gives "Error and problem log". Does this translation include the fact that "issues" might be enhancement or feature requests, too?

Also now that I think of it, both Polish and German translations should probably mention that these links are all exclusively for English. Or I guess we could add a pl and de chat on Discord if people want that, but I want to keep Github in English, and my Twitter DMs should also be either English or German.